src/main/java/org/cactoos/set/Sorted.java

```diff
@@ -54,7 +54,10 @@ public Sorted(final Comparator<T> cmp, final T... array) {
      * @param src An {@link Iterable}
      */
     @SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
-    public Sorted(final Comparator<T> cmp, final Iterable<T> src) {
+    public Sorted(
+        final Comparator<T> cmp,
```

Comment:
@andreoss here there is still one comparator without the `? super T`

Reply:
@victornoel Done

Reply:
@andreoss I still don't see it in the PR's diff...